lib/charms/opensearch/v0/opensearch_config.py

```diff
@@ -69,7 +69,7 @@ def set_admin_tls_conf(self, secrets: Dict[str, any]):
         self._opensearch.config.put(
             self.CONFIG_YML,
             "plugins.security.authcz.admin_dn/{}",
-            f"{normalized_tls_subject(secrets['subject'])}",
+            f"{normalized_tls_subject(secrets.get('subject'))}",
```

Comment:
What if `secrets.get('subject')` returns `None`? What does `normalized_tls_subject(secrets.get('subject'))` return in that case?

Shouldn't we first check if there's a valid (not `None`, not "", etc.) value, and _then_ `put()` the value to the config?

Reply:
This is handled in the base charm module with `if not admin_secrets.get("subject"):` (the one you commented on above).

Reply:
Done